Evaluate-ModelarDB-Compression/main.py

```diff
@@ -0,0 +1,281 @@
+""" Script for evaluating ModelarDB with different data sets. """
+
+
+import sys
+import math
+import time
+import signal
+import tempfile
+import subprocess
+
+import numpy
+from pyarrow import parquet
+from pyarrow import flight
+
+
+# Configuration.
+MODELARDB_REPOSITORY = "https://github.com/ModelarData/ModelarDB-RS.git"
+UTILITIES_REPOSITORY = "https://github.com/ModelarData/Utilities.git"
+TABLE_NAME = "evaluate"
+STDOUT = subprocess.PIPE
+STDERR = subprocess.PIPE
+
+
+# Helper Functions.
+def extract_repository_name(url):
+    return url[url.rfind("/") + 1 : url.rfind(".")] + "/"
+
+
+def git_clone(url):
+    subprocess.run(["git", "clone", url], stdout=STDOUT, stderr=STDERR)
+
+
+def cargo_build_release(modelardb_folder):
```

Comment:
This might be overkill for a utility repository but it seems that there is quite a bit of duplicated code between this file and `Evaluate-ModelarDB-Changes/main.py`. Maybe create a utility file with the common functionality.

Reply:
I also came to the conclusion that it is overkill for now due to the number of and size of scripts we currently have. However, I do think it will become a problem if we continue adding more scripts and extend the existing scripts. Thus, I have created a low priority task to move shared functionality into a separate shared Python module.